src/dft/src/cells/OneBitScanCell.hh

```diff
@@ -59,6 +59,9 @@ class OneBitScanCell : public ScanCell
   void connectScanOut(const ScanLoad& load) const override;
   ScanDriver getScanOut() const override;
 
+  odb::Point getOrigin() const override;
```

Comment:
odb::Point is a class, I think it is a good idea to return a const reference to it:

```suggestion
  const odb::Point& getOrigin() const override;
```

Reply:
The idiom in OpenROAD elsewhere (e.g. the `dbInst::getPoint` method it wraps) is to just pass it by value, being a 64 bit object (2 integers) anyway there's no real benefit to using a reference.